cmeutils/tests/test_plot.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+import scipy.signal as signal
+import numpy as np
+
+from base_test import BaseTest
+
+from cmeutils.plot_tools import find_peaks
+
+class TestPlot(BaseTest):
+    def test_plot(self):
+        x = [1,2,3,4,5]
```

Comment:
In order to design a more robust test, let's test your peak finder function on some real data where we know where the peaks should be. (What you have now is a great start--it's never a bad idea to test that your output data is the correct type!) Here's how I would go about this:
- Find an RDF with some nice looking peaks
- Save/commit the RDF to a txt file in the `tests/assets` folder
- Within your test, load up the RDF and use `find_peaks` on the y-values. It can be a little wonky to point to the assets directory--you can either make an [rdf fixture for BaseTest](https://docs.pytest.org/en/6.2.x/fixture.html) (There are other examples in BaseTest) or use the `os` library to point to the asset dir like [this](https://github.com/cmelab/cmeutils/blob/2750eac559fd04e9e8354132e6032078d56dd572/cmeutils/tests/base_test.py#L9).
- Use your test to check that the peaks are at the right spots and that the correct number of peaks are found!

Don't hesitate to reach out if any of that doesn't make sense or you get stuck!

Reply:
This sounds great, I'll do this. Thanks Jenny!